release-notes.txt

```diff
@@ -1,3 +1,634 @@
+RIOT-2021.01 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2021.01 release includes:
+ - Device Firmware Upgrade support for USBUS
+ - USB HID minimal implementation
+ - IEEE 802.15.4 security initial implementation
+ - 6LowPAN Minimal Forwarding and Selective Fragment Recovery initial
+   implementation
+ - Initial support for QN808x, SAMD10 STM32MP1 and STM32L5 CPUs
+ - Initial support for Cortex-M33
+ - A module for selecting entropy sources
+ - Initial support for the Decawave Impulse Radio-Ultra Wideband (UWB Core)
+ - Capability to join or leave link layer multicast groups (GNRC)
+ - Support for IN865 region in GNRC LoRaWAN
+ - 83 additional modules supported by Kconfig
+ - 14 new boards, 6 new device drivers, 8 packages updated
+
+In order to improve community interaction, during this release cycle we started
+to migrate from a mailing-list-based communication to forum-based
+communication. We are proud to present the RIOT-OS forum available at
+forum.riot-os.org
+
+533 pull requests, composed of 1440 commits, have been merged since the
+last release, and 65 issues have been solved. 58 people contributed with
+code in 119 days. 3852 files have been touched with 348619 (+) insertions and
+52516 deletions (-).
+
+Notations used below:
+=====================
+
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+New features and changes
+========================
+
+Core
+----
++ msg_bus: allow to differ between messages form bus and from thread
++ mutex: Add mutex_cancel
++ mbox: add mbox_size(), mbox_avail()
+* mutex: compatibility with non-C languages
```

Comment:
```suggestion
```

This was a bug fix to an issue that popped up during the development of the release cycle. Compared to old releases, compatibility with non-C languages remained identical. I think it might be less confusing to just drop this.

Reply:
removed :)